### PR TITLE
clean tabs after paragrafs

### PR DESCRIPTION
**Pull request by windreis**

### CODE REVIEW

**Comment by NelsonBN** [comment]
Hello jose... This is a very good PR 👍

File: index.html [hunk 1]
```diff
@@ -12,10 +12,10 @@
     </p>
     <p>
         flçdkfdçlkfdl
-    </p>    
+    </p>
```
Comment (NelsonBN) [inline index.html]: I think is good idea also add <br>

**Review by NelsonBN — CHANGES_REQUESTED** [review]
Please make my suggestions

File: index.html (hunk 1 above)
Comment (windreis) [reply index.html]: Good point, already added br

**Review by NelsonBN — APPROVED** [review]
Very nice!!!!!

**Review by NelsonBN — APPROVED** [review]

**Review by NelsonBN — APPROVED** [review]

**Review by NelsonBN — APPROVED** [review]

**Review by NelsonBN — APPROVED** [review]

**Pull request merged**